rwgame/main.cpp

```diff
@@ -1,9 +1,23 @@
 #include "RWGame.hpp"
+#include "SDL.h"
+#include <iostream>
 
-int main(int argc, char *argv[])
+int main(int argc, char* argv[])
 {
+	try {
+		RWGame game(argc, argv);
 
-	RWGame game( argc, argv );
+		return game.run();
+	} catch (std::exception& ex) {
+		const char* kErrorTitle = "Fatal Error";
 
-	return game.run();
+		std::cerr << kErrorTitle << "\n" << ex.what() << std::endl;
```

Comment:
I did think about that, and improving logging is something we should do for usability if we want to give more useful error messages in general. It's beyond this PR for sure.